src/components/Pokemon/Pokemon.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import './Pokemon.css';
+
+export default function Pokemon({ poke }) {
+  const { url_image, pokemon, attack, defense, hp, type_1, type_2 } = poke;
```

Comment:
you can spread your props in and eliminate this destructuring step like so: `<Pokemon poke={...pokemon} />` (this would happen in the `map` in `List.jsx`)